packages/terra-dialog/src/Dialog.module.scss

```diff
@@ -30,11 +30,16 @@
 
   .close-icon {
     background: var(--terra-dialog-header-close-background, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='rgba(17, 17, 17, 1)' d='M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z'/></svg>"));
+    background-repeat: no-repeat;
+    background-size: auto;
     display: inline-block;
-    height: var(--terra-dialog-header-close-icon-height, 1.14286rem);
-    margin-top: var(--terra-dialog-header-close-icon-margin-top, -0.214285rem);
-    vertical-align: middle;
-    width: var(--terra-dialog-header-close-icon-width, 1.14286rem);
+    height: var(--terra-dialog-header-close-icon-height, 1rem);
+    margin-top: var(--terra-dialog-header-close-icon-margin-top, 0);
+    vertical-align: var(--terra-dialog-header-close-icon-vertical-align, -0.15rem);
+    width: var(--terra-dialog-header-close-icon-width, 1rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      vertical-align: var(--terra-dialog-header-close-icon-vertical-align, -0.15rem);
```

Comment:
@neilpfeiffer @saket2403 ,
Do we need to differentiate this theme variable from the normal vertical-align theme variable ([here](https://github.com/cerner/terra-core/pull/2523/files?file-filters%5B%5D=.scss#diff-29f4c6651a36c677ff2dcc4d9a2770d6R38)). 

Something similar to as seen [here](https://github.com/cerner/terra-core/pull/2523/files?file-filters%5B%5D=.scss#diff-b2ae2172c829371a523555d2e0626b06R18-R21)